### ISSUE TITLE
Request: Add option to uninstall

### ISSUE THREAD

**Issue by MyNameIsMohamad** [dagrha/pypia#27]
Hello dagrha,

It would be most appreciated by me and (hopefully) other users that you add an uninstall option to pypia when installing via manual method (git clone method). If not, add maybe an uninstall guide in the README.

Much appreciated,
Mohamad

**Comment by dagrha** [comment]
Thanks for the comment, Mohamad. I can look into a good uninstall routine. In the meantime, I'd love to hear your feedback about why you want to uninstall it. Does pypia not work for you? Something bad about the user experience that is driving you to uninstall? Let me know and I can try to improve it.

You can remove the keys files by using this command:
`find /etc/NetworkManager/system-connections/ -name 'PIA*' -delete`

To delete the PIA cert you can do `rm /etc/openvpn/ca.rsa.2048.crt`

Then you can just delete the folder you cloned. 

That should take care of all the files that pypia adds.

**Comment by MyNameIsMohamad** [comment]
dagrha,

I simply want to uninstall it so that I can install it through pip. I have followed through with your instructions and they have completed successfully.

To mention, on Fedora 27 the 'pip' command uses python 2.7 while 'pip-3', 'pip3.6' 'pip-3.6' 'pip3' use python3. I remember trying to install pypia with the 'pip' command but I got a bunch of errors, presumably because I was using the 'pip' command, which uses python 2.7.

Best regards,
Mohamad
